nixos/modules/services/security/fail2ban.nix

```diff
@@ -205,6 +210,15 @@ in
           defined in <filename>/etc/fail2ban/action.d</filename>,
           while filters are defined in
           <filename>/etc/fail2ban/filter.d</filename>.
+
+          NixOS comes with a default <literal>sshd</literal> jail;
+          for it to work well, you should set
```

Comment:
The wording makes it sound like action is required here under any scenario.

Reply:
I have improved the wording, switching it to a passive to make it clearer.

Reply:
The documentation still says I have to do something when I don't. The `fail2ban` takes care of it for me, but documentation says I have to.

Reply:
@aanderse Right in the next sentence it says

```
          This module sets it to <literal>"VERBOSE"</literal> if
          not set otherwise, so enabling fail2ban can make SSH logs
          more verbose. 
```

indicating that you not have to do something unless you have otherwise set `services.openssh.logLevel`.

Reply:
Need more sleep :sleeping: Sorry for the hassle :wink:

Reply:
No prob :grin: